java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -1218,16 +1069,9 @@ public void shouldInitialiseRootPartitionCorrectlyForLongKey() throws StateStore
 
         // Then
         assertThat(partitions).hasSize(1);
```

Comment:
We could remove this assertion, as containsExactly already asserts this with better output.